packages/core/utils/lib/index.js

```diff
@@ -4,11 +4,7 @@
  * Export shared utilities
  */
 const { buildQuery, hasDeepFilters } = require('./build-query');
-const {
-  convertRestQueryParams,
```

Comment:
Why is it not exported anymore? The id is to migrate the code directly in Strapi package?

Reply:
Just because I deleted the function but I'll refactor the code again once the content api is a bit more ready